resources/integration_test/tests/functions.yml

```diff
@@ -92,35 +92,35 @@ tests:
-      - [ "binData", -9, null, null, null, null, 1, 0, 0, null, 0, null, "binData", null, null, -9, null, null, null ]
-      - [ "binData", -9, null, null, null, null, 1, 0, 0, null, 0, null, "binData", null, null, -9, null, null, null ]
-      - [ "string", -1, 65535, "'", "'", null, 1, 1, 3, null, 0, null, "string", null, null, -1, null, null, null ]
-      - [ "array", -9, null, null, null, null, 1, 0, 0, null, 0, null, "array", null, null, -9, null, null, null ]
-      - [ "bson", -9, null, null, null, null, 1, 0, 0, null, 0, null, "bson", null, null, -9, null, null, null ]
-      - [ "dbPointer", -9, null, null, null, null, 1, 0, 2, null, 0, null, "dbPointer", null, null, -9, null, null, null ]
-      - [ "decimal", -9, null, null, null, null, 1, 0, 2, 0, 0, 0, "decimal", null, null, -9, null, null, null ]
-      - [ "javascript", -9, null, null, null, null, 1, 0, 2, null, 0, null, "javascript", null, null, -9, null, null, null ]
-      - [ "javascriptWithScope", -9, null, null, null, null, 1, 0, 2, null, 0, null, "javascriptWithScope", null, null, -9, null, null, null ]
-      - [ "maxKey", -9, 14, null, null, null, 1, 0, 2, null, 0, null, "maxKey", null, null, -9, null, null, null ]
-      - [ "minKey", -9, 14, null, null, null, 1, 0, 2, null, 0, null, "minKey", null, null, -9, null, null, null ]
-      - [ "null", -9, 4, null, null, null, 1, 0, 0, null, 0, null, "null", null, null, -9, null, null, null ]
-      - [ "object", -9, null, null, null, null, 1, 0, 0, null, 0, null, "object", null, null, -9, null, null, null ]
-      - [ "objectId", -9, 35, null, null, null, 1, 0, 2, null, 0, 1, "objectId", null, null, -9, null, null, null ]
-      - [ "symbol", -9, null, null, null, null, 1, 0, 2, null, 0, null, "symbol", null, null, -9, null, null, null ]
-      - [ "timestamp", -9, 68, null, null, null, 1, 0, 2, null, 0, null, "timestamp", null, null, -9, null, null, null ]
-      - [ "undefined", -9, 20, null, null, null, 1, 0, 0, null, 0, null, "undefined", null, null, -9, null, null, null ]
-      - [ "string", 1, 65535, "'", "'", null, 1, 1, 3, null, 0, null, "string", null, null, 1, null, null, null ]
-      - [ "int", 4, 10, null, null, null, 1, 0, 2, 0, 1, 0, "int", 0, 0, 4, null, 10, null ]
-      - [ "int", 5, 10, null, null, null, 1, 0, 2, 0, 1, 0, "int", 0, 0, 5, null, 10, null ]
-      - [ "double", 7, 15, null, null, null, 1, 0, 2, 0, 0, 0, "double", 0, 0, 7, null, 2, null ]
-      - [ "double", 8, 15, null, null, null, 1, 0, 2, 0, 0, 0, "double", 0, 0, 8, null, 2, null ]
-      - [ "date", 11, 23, "'", "'", null, 1, 0, 2, null, 1, null, "date", 3, 3, 11, 3, null, null ]
-      - [ "varchar", 12, 65535, "'", "'", null, 1, 1, 3, null, 0, null, "varchar", null, null, 12, null, null, null ]
-      - [ "date", 93, 23, "'", "'", null, 1, 0, 2, null, 1, null, "date", 3, 3, 93, 3, null, null ]
+    - [ "string", -10, 65535, "'", "'", null, 1, 1, 3, null, 0, null, "string", null, null, -10, null, null, null ]
```

Comment:
Sorry about the space deleting here, but it was driving me crazy that this test didn't line up with the other tests